torch_geometric/nn/aggr/multi.py

```diff
@@ -2,23 +2,28 @@
 
 import torch
 from torch import Tensor
+from torch.nn import Linear
+from torch_scatter import scatter
 
 from torch_geometric.nn.aggr import Aggregation
 from torch_geometric.nn.resolver import aggregation_resolver
 
 
 class MultiAggregation(Aggregation):
     def __init__(self, aggrs: List[Union[Aggregation, str]],
-                 aggrs_kwargs: Optional[List[Dict[str, Any]]] = None):
+                 aggrs_kwargs: Optional[List[Dict[str, Any]]] = None,
+                 combine_mode: str = 'cat', in_channels: Optional[int] = None,
+                 out_channels: Optional[int] = None):
+
```

Comment:
Can we add a docstring? I think it would be good to describe how the combine mode works here

Reply:
Thanks. Will add it.